specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2020-04-01-preview/providers.json

```diff
@@ -152,6 +159,81 @@
           "description": "Blob duration of specification."
         }
       }
+    },
+    "MetricSpecification": {
+      "description": "Metric specification of operation.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of metric specification."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "Display name of Metric specification."
+        },
+        "displayDescription": {
+          "type": "string",
+          "description": "Display description of Metric specification."
+        },
+        "unit": {
+          "type": "string",
+          "description": "The metric unit. Possible values include: 'Bytes', 'Count', 'Milliseconds'."
+        },
+        "aggregationType": {
+          "type": "string",
+          "description": "The metric aggregation type. Possible values include: 'Average', 'Count', 'Total'."
+        },
+        "supportedAggregationTypes": {
+          "type": "array",
+          "description": "The supported aggregation types for the metrics.",
```

Comment:
what's difference between `supportedAggregationTypes` and `aggregationType` ? Will it be clear to define `aggregationType` as enum definition, then type of `supportedAggregationTypes` be `aggregationType`?

Reply:
As far as I know, SupportedAggregationTypes is a composite of the Saturation metric as insights does not have a mechanism in which we can specify which sampling type to present to the customer.

This PR is to fix swagger quality issue in s360. These two properties are already defined in our source code. It should not be a good way to update the source code at this moment.

Reply:
thanks for the context